pages/linux/dnf-automatic.md

```diff
@@ -0,0 +1,16 @@
+# dnf-automatic
+
+> A tool for automating the checking, downloading, and/or applying package updates via dnf.
+> More information: <https://dnf.readthedocs.io/en/latest/automatic.html>.
+
+- Override config file and only notify of available updates:
+
+`sudo systemctl enable --now dnf-automatic-notifyonly.timer`
+
+- Run with default config file:
+
+`sudo systemctl enable --now dnf-automatic.timer`
+
+- Use dnf-automatic in a custom systemd timer:
+
+`/usr/bin/dnf-automatic /path/to/automatic.conf --timer`
```

Comment:
This command is already in the path :thinking: 
```suggestion
`dnf-automatic {{path/to/automatic.conf}} --timer`
```

Reply:
The [systemd.service unit file ExecStart](https://www.freedesktop.org/software/systemd/man/systemd.service.html#ExecStart=) requires absolute paths to binaries.

Would this be clearer?

```suggestion
`ExecStart=/usr/bin/dnf-automatic {{path/to/automatic.conf}} --timer`
```

Reply:
If it requires full paths, then we prefix with `/`.
```suggestion
`/usr/bin/dnf-automatic {{/path/to/automatic.conf}} --timer`
```

While the `ExecStart=` is useful in a systemd service file etc, it should not be present when you actually execute the command, so it shouldn't be included in a tldr page.

Reply:
> If it requires full paths, then we prefix with `/`.
> 
> While the `ExecStart=` is useful in a systemd service file etc, it should not be present when you actually execute the command, so it shouldn't be included in a tldr page.

Ah, good point! Thank you